packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -285,6 +294,9 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
     ) {
       this._setBeakPosition();
     }
+    if (prevProps.preventDismissOnLostFocus !== this.props.preventDismissOnLostFocus) {
+      this._addListeners();
```

Comment:
if you call this again, you'll add more events without clearing existing. you want to call `this._events.off()` from inside `_addListeners`, and then you can certainly call it again without worrying about double initialization.